analytics.py

```diff
@@ -30,5 +37,5 @@ def __init__(self, _db):
     def collect_event(self, library, license_pool, event_type, time=None, **kwargs):
         if not time:
             time = datetime.datetime.utcnow()
-        for provider in self.providers:
+        for provider in (self.sitewide_providers + self.library_providers[library.id]):
```

Comment:
That's a good point. If there's a collection with multiple libraries it will cause duplicate events in the sitewide provider. I was thinking that I'd have to add the library id to the CirculationEvents table and the Google Analytics setup in a future pr though.